client/db/test/dbtest.go

```diff
@@ -223,7 +223,7 @@ func MustCompareWallets(t testKiller, w1, w2 *db.Wallet) {
 	if w1.Balance != w2.Balance {
 		t.Fatalf("Balance mismatch. %d != %d", w1.Balance, w2.Balance)
 	}
-	if !w1.BalUpdate.Equal(w2.BalUpdate) {
+	if w1.BalUpdate.Unix() != w2.BalUpdate.Unix() {
```

Comment:
To compare with millisecond resolution, you could Truncate each, or compare the `int64`s from `encode.UnixMilli`, which would be in line with most of the rest of the codebase.